app/scripts/proactive/view/dom.js

```diff
@@ -422,17 +422,37 @@ define(
             var jobGenericInfos = studioApp.models.jobModel.get("Generic Info");
 
 
-            var jobVariables = readOrStoreVariablesInModel();
-            if (jobVariables == null || $.isEmptyObject(jobVariables)) {
-                executeIfConnected(submit);
-                return;
-            }
+            var jobVariablesOriginal = readOrStoreVariablesInModel();
+
+            var jobVariables = {};
 
             var validationData = validate();
-            if (validationData.updatedModels != null) {
-                for( var key in jobVariables){
-                    jobVariables[key].resolvedModel = validationData.updatedModels[key];
+            if (validationData.updatedModels != null && validationData.updatedVariables != null) {
```

Comment:
it can be `if (validationData.updatedModels and validationData.updatedVariables)`